transport/cluster.go

```diff
@@ -0,0 +1,268 @@
+package transport
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"strconv"
+	"sync/atomic"
+
+	"scylla-go-driver/frame"
+	. "scylla-go-driver/frame/request"
+	. "scylla-go-driver/frame/response"
+)
+
+var (
+	peerQuery = Statement{
+		Content:     "SELECT peer, data_center, rack, tokens FROM system.peers",
+		Consistency: frame.ONE,
+	}
+
+	localQuery = Statement{
+		Content:     "SELECT rpc_address, data_center, rack, tokens FROM system.local",
+		Consistency: frame.ONE,
+	}
+)
+
+func makePeerStatement(addr frame.Inet) Statement {
```

Comment:
How about peersQuery?

Reply:
Wouldn't it be confusing to name it almost the same as the above var peerQuery?